mysql-test/suite/galera/t/galera_ftwrl_concurrent.test

```diff
@@ -0,0 +1,61 @@
+#
+# MDEV-32282
+#
+# A node remains in paused state after two interleaving FTWRLs,
+# and the following CREATE TABLE fails with
+#
+#  ER_UNKNOWN_COM_ERROR (1047): Aborting TOI: Replication paused on
+#  node for FTWRL/BACKUP STAGE.
+#
+# node_1                                         node_1_a
+# ----------------------------------------------------------------------
+# FTWRL
```

Comment:
Can you add case with BACKUP STAGE as variable wsrep_desynced_backup_stage is also used there.